guideline-03.md

```diff
@@ -1,5 +1,5 @@
 <h4>3. A stable version of a plugin must be available from its WordPress Plugin Directory page.</h4>
 
-The only version of the plugin that WordPress.org distributes is the one in the directory. Though people may develop their code somewhere else, users will be downloading from the directory, not the development environment.
+The only version of the plugin that WordPress.org distributes is what is hosted in the directory. Authors may develop their code where they wish, but users will only download the completed product from the WordPress.org directory.
```

Comment:
Does this mean that authors can't even offer the plugin as a direct download from a location their own website and has to ALWAYS link back to wordpress.org?

Reply:
No. It means that you can't use the plugin hosted here to install from anywhere outside of WordPress.org. Though I would seriously question the developer who wants to go through the drama and work of uploading the zip in two places when WordPress.org handles the updates etc for them.